github/repos.go

```diff
@@ -283,7 +286,9 @@ func (s *RepositoriesService) Get(owner, repo string) (*Repository, *Response, e
 
 	// TODO: remove custom Accept header when the license support fully launches
 	// https://developer.github.com/v3/licenses/#get-a-repositorys-license
+	// https://developer.github.com/changes/2016-09-26-pull-request-merge-api-update
```

Comment:
The blog posts are usually referenced in the github.go file with the preview tags - you could add this to the `polaris` one.